### PR TITLE
Get matplotlib version old-fashioned way

### PR DESCRIPTION
**Pull request by raoulcollenteur**
# Short Description
ps.show_versions is returning None for MPL, fix that.

# Checklist before PR can be merged:
- [X] closes issue #704 
- [ ] Format code with [Black formatting](https://black.readthedocs.io)
- [ ] tests added / passed

### CODE REVIEW

File: pastas/version.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import logging
 from importlib import import_module, metadata
 from platform import python_version
+from matplotlib import __version__ as mplversion
+
+mplversion
```
Comment (martinvonk) [inline pastas/version.py:6]: remove line

File: pastas/version.py [hunk 2]
```diff
@@ -44,7 +47,7 @@ def show_versions(lmfit: bool = True, latexify: bool = True) -> None:
         f"NumPy version: {metadata.version('numpy')}\n"
         f"Pandas version: {metadata.version('pandas')}\n"
         f"SciPy version: {metadata.version('scipy')}\n"
-        f"Matplotlib version: {metadata.version('matplotlib')}\n"
+        f"Matplotlib version: {mplversion}\n"
```
Comment (martinvonk) [inline pastas/version.py:50]: maybe:     
mplversion= metadata.version("matplotlib") if metadata.version("matplotlib") is not None else mplversion

**Comment by martinvonk** [comment]
Not sure why this fails, maybe related to the matplotlib version? Or something else? #704

**Comment by raoulcollenteur** [comment]
I have version 3.8.0 and it shows None.. and now I checked another environment and for 3.8.3 it does work :p 

So I'll close this as I think MPL fixed it. Not worth the hassle.

**Pull request closed without merging**